shell/app/common/components/key-value-table.tsx

```diff
@@ -270,7 +268,7 @@ export class KeyValueTable extends React.Component<IProps, IState> {
       {
         title: 'KEY',
         dataIndex: 'key',
-        width: '40%',
+        width: 296,
```

Comment:
I have one question. In the `WrappedTable` `width` is omitted. Then what's the point of passing `width` prop?

Reply:
I made a mistake in WrappedTable. I wanted to add a size attribute instead of width, but I gave up.